core/server/api/themes.js

```diff
@@ -26,11 +27,24 @@ var debug = require('debug')('ghost:api:themes'),
 themes = {
     browse: function browse() {
         debug('browsing');
-        var result = packageUtils.filterPackages(themeList.getAll());
+        var result = themeList.toAPI(themeList.getAll(), settingsCache.get('activeTheme'));
```

Comment:
100% agree though this is weird and fragile, all I'm trying to say is it's a product of partial refactoring and I totally WILL change all of this v. soon.

Next on my list: simplifying/unifying the logic around activating.

This PR makes that MUCH easier cos it's no longer dependent on a settings change 😁